src/integrationTest/sam.test.ts

```diff
@@ -80,7 +80,7 @@ async function getAddConfigCodeLens(documentUri: vscode.Uri): Promise<vscode.Cod
                 return codeLenses[0]
             }
         } catch (e) {
-            console.log(`getAddConfigCodeLens(): failed, retrying:\n${e}`)
+            console.log(`sam.test.ts: getAddConfigCodeLens(): failed, retrying:\n${e}`)
```

Comment:
Are these `console.log` steps necessary?

Reply:
yes, they add info to the CI logs